modules/prebidServerBidAdapter.js

```diff
@@ -19,15 +19,39 @@ const DEFAULT_S2S_CURRENCY = 'USD';
 const DEFAULT_S2S_NETREVENUE = true;
 
 let _s2sConfig;
+
+const s2sDefaultConfig = {
+  enabled: false,
+  timeout: 1000,
+  maxBids: 1,
+  adapter: 'prebidServer'
+};
+
 config.setDefaults({
-  's2sConfig': {
-    enabled: false,
-    timeout: 1000,
-    maxBids: 1,
-    adapter: 'prebidServer'
-  }
+  's2sConfig': s2sDefaultConfig
 });
 
+// accountId and bidders params are not included here, should be configured by end-user
+const availVendorDefaults = {
+  'appnexus': {
+    adapter: 'prebidServer',
+    cookieSet: false,
+    enabled: true,
+    endpoint: '//prebid.adnxs.com/pbs/v1/auction',
+    syncEndpoint: '//prebid.adnxs.com/pbs/v1/cookie_sync',
+    timeout: 1000
+  },
+  'rubicon': {
+    adapter: 'prebidServer',
+    cookieSet: true,
+    cookieSetUrl: 'https://secure-assets.rubiconproject.com/utils/cookieset/cs.js',
```

Comment:
We discussed -- please remove the cookieSetUrl for Rubicon and turn cookieSet to false. Thanks. Also make "endpoint" protocol-relative. Thanks.